integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/viewpopenqueueservice/test/ViewPopEnqueueTest.java

```diff
@@ -0,0 +1,120 @@
+package org.wso2.carbon.esb.viewpopenqueueservice.test;
+
+import org.wso2.carbon.automation.extensions.servers.jmsserver.client.JMSQueueMessageProducer;
+import org.wso2.carbon.automation.test.utils.http.client.HttpRequestUtil;
+import org.wso2.carbon.message.processor.stub.MessageProcessorAdminService;
+import org.wso2.carbon.message.store.stub.MessageInfo;
+import org.wso2.esb.integration.common.clients.mediation.MessageStoreAdminClient;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * Tests to verify the functionality
+ *
+ */
+
+public class ViewPopEnqueueTest extends ESBIntegrationTest {
+
+    private MessageStoreAdminClient messageStoreAdminClient;
+    MessageProcessorAdminService messageProcessorAdminService;
+    private JMSQueueMessageProducer jmsQueueMessageProducer;
+
+    private final String STORE_NAME = "VPE_Store";
+    private final String PROCESSOR_NAME = "VPE_Processor";
+    private final String PROXY_NAME = "VPE_Proxy";
+
+    /**
+     *  Initializing environment variables
+     */
+
+    @BeforeClass(alwaysRun = true, description = "Test Message store View, Pop and Enqueue")
+    protected void setup() throws Exception {
+        super.init();
+
+        loadESBConfigurationFromClasspath("artifacts/ESB/messageProcessorConfig/viewPopEnqueueTestConfig.xml");
+
+        // Initialization
+        messageStoreAdminClient = new MessageStoreAdminClient(context.getContextUrls().getBackEndUrl(), sessionCookie);
+
+        // Verifying Existence
+        verifyMessageStoreExistence(STORE_NAME);
+        verifyMessageProcessorExistence(PROCESSOR_NAME);
+        isProxySuccesfullyDeployed(PROXY_NAME);
+        //verifySequenceExistence(PROXY_NAME);
+
+    }
+
+    /**
+     *  1. Invoke the proxy with one request while the backend is unavailable
+     *  2. Verify the existence of messages in message store + automatic de-activation of processor due to backend failure
+     *  3. Call getMessage function and verify that the queue is sending the right message
+     *
+     *  Run View and Pop functions
+     *  Assert
+     */
+
```

Comment:
Let's remove all unnecessary new lines

Reply:
This test has been removed from the PR and no longer exists